stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.0.4
+version: 2.0.5
```

Comment:
Please bump minor version as this brings a new feature.

Reply:
Now set to 2.1.0